[2024-08-17T16:21:58.925+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl.obtain_data manual__2024-08-17T15:47:53.714343+00:00 [queued]>
[2024-08-17T16:21:58.930+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl.obtain_data manual__2024-08-17T15:47:53.714343+00:00 [queued]>
[2024-08-17T16:21:58.930+0000] {taskinstance.py:2170} INFO - Starting attempt 5 of 6
[2024-08-17T16:21:58.938+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): obtain_data> on 2024-08-17 15:47:53.714343+00:00
[2024-08-17T16:21:58.942+0000] {standard_task_runner.py:60} INFO - Started process 670 to run task
[2024-08-17T16:21:58.944+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'etl', 'obtain_data', 'manual__2024-08-17T15:47:53.714343+00:00', '--job-id', '18', '--raw', '--subdir', 'DAGS_FOLDER/etl_process.py', '--cfg-path', '/tmp/tmprek9gc3g']
[2024-08-17T16:21:58.945+0000] {standard_task_runner.py:88} INFO - Job 18: Subtask obtain_data
[2024-08-17T16:21:58.976+0000] {task_command.py:423} INFO - Running <TaskInstance: etl.obtain_data manual__2024-08-17T15:47:53.714343+00:00 [running]> on host ab9e9042f5d0
[2024-08-17T16:21:59.026+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl' AIRFLOW_CTX_TASK_ID='obtain_data' AIRFLOW_CTX_EXECUTION_DATE='2024-08-17T15:47:53.714343+00:00' AIRFLOW_CTX_TRY_NUMBER='5' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-17T15:47:53.714343+00:00'
[2024-08-17T16:22:03.831+0000] {logging_mixin.py:188} INFO - -----------------------Datos guardados en PostgreSQL en la tabla: data
[2024-08-17T16:22:03.832+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-08-17T16:22:03.838+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=etl, task_id=obtain_data, execution_date=20240817T154753, start_date=20240817T162158, end_date=20240817T162203
[2024-08-17T16:22:03.892+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-08-17T16:22:03.904+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
